### ISSUE TITLE
Color handling is a little confusing (rendering in the app and in super simple export seems like it may be incorrect)

### ISSUE THREAD

**Issue by bcolloran** [deepnight/ldtk#887]
Thank you so much for LDTK, it's really a joy to use! I've encountered what I think is a bug in LDTK's color handling; if it's not a bug, then perhaps the options for dealing with compositing just need a little UX love :-)

In the screenshot below, you can see (easier to see if you open the image at large size in a new tab):
- left: the LDTK window showing a map using the cavernas tilemap
- my OS image previewer rendering of the original cavernas tilemap file
- my OS image previewer rendering the `_composite.png` file created by "super simple export"

![image](https://user-images.githubusercontent.com/2380975/231065494-8a3f9105-74a5-48d6-8437-6a820c09b253.png)

As you can see in the image, the colors in the LDTK window don't match the colors used in the tileset -- in LDTK they look a little muted and washed out. Further, the `_composite.png` doesn't match what is shown in LDTK -- the colors are even darker, and the background color is not applied in the same way.

I've played around with all the background color options I can find (project, app, and world), but I can't get consistent results. It also seems like the background color is applied on top of the tiles, or that the tiles are somehow made transparent, as in this screenshot where I've set the background to be red, but the entire map ends up tinted:
![image](https://user-images.githubusercontent.com/2380975/231067969-890cae32-401d-4088-876e-2ded64614ce1.png)
(I've checked in Asesprite and the alpha channel in the tilemap is set to be entirely opaque, so the backgroumd should not be showing through the tiles)

I'm not sure whether I'm doing something wrong (if there is some additional setting tucked away that I did not see) or if there is a color handling bug somewhere.

Thanks!

**Comment by deepnight** [comment]
I could not reproduce the bug at all :( 

Have you any specific setup outside of LDtk?

**Comment by bcolloran** [comment]
Thanks for looking at this @deepnight!

Since you couldn't reproduce the bug, I tried deleting and re-downloading the ldtk 1.25 binary (direct from the GH versions page)... and it's working fine now!... ¯\_(ツ)_/¯

I'll go ahead and close this issue since it seems hard to reproduce the bug, but for future reference here is the general setup where I encountered the funny behavior:
- LDtk 1.25
- Release Linux Mint 20.1 Ulyssa 64-bit
- Kernel Linux 5.4.0-146-generic x86_64
- MATE 1.24.0
- AMD Ryzen 5 2600 Six-Core Processor × 12
- NVIDIA GeForce RTX 2060/PCIe/SSE2

Happy to provide any more info if you want to look into this further, but no need on my part. I guess the other takeaway is to just try a fresh copy of the binary if things are acting strange.

Thanks again for LDtk! Bought it on itch a while back, still loving it! :-)
